### PR TITLE
Update to liberty-maven-plugin-2.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by josephca**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -113,59 +81,41 @@
...
+                    <target>1.7</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>2.7</version>
+                <executions>
+                    <execution>
+                        <id>copy-app</id>
```
Comment (patricktiu) [inline pom.xml]: Do we still need to use maven-resource-plugin to copy the app?

File: pom.xml [hunk 2]
```diff
@@ -185,13 +135,13 @@
...
 				<artifactId>liberty-maven-plugin</artifactId>
-				<version>1.2</version>
+				<version>2.0-SNAPSHOT</version>
 				<extensions>true</extensions>
 				<configuration>
 					<assemblyArtifact>
 						<groupId>com.ibm.websphere.appserver.runtime</groupId>
-						<artifactId>wlp-javaee7</artifactId>
-						<version>16.0.0.4</version>
+						<artifactId>wlp-webProfile7</artifactId>
```
Comment (patricktiu) [inline pom.xml]: Can you change from a JEE 7 Full profile runtime to a JEE 7 Web Profile runtime?  It seems that jaxws-2.2 is not included in WebProfile7 runtime.

File: pom.xml [hunk 3]
```diff
@@ -202,12 +152,12 @@
...
 						<default.http.port>${testServerHttpPort}</default.http.port>
 						<default.https.port>${testServerHttpsPort}</default.https.port>
-						<appLocation>${project.artifactId}.war</appLocation>
+						<appLocation>${project.build.finalName}.war</appLocation>
 					</bootstrapProperties>
 				</configuration>
 				<executions>
 					<execution>
-						<id>intall-liberty</id>
+						<id>install-liberty</id>
```
Comment (patricktiu) [inline pom.xml]: install-liberty is already defined in parent pom.

**Comment by patricktiu** [comment]
Do you think we need to delete .factorypath file from the repository?

File: pom.xml (hunk 2 above)
Comment (josephca) [inline pom.xml]: Change back to wlp-javaee7.

File: pom.xml (hunk 1 above)
Comment (josephca) [inline pom.xml]: Yes, it seems that it requires to copy the app to 'servers/defaultServer/apps'

File: pom.xml (hunk 3 above)
Comment (josephca) [inline pom.xml]: Removed.

**Comment by josephca** [comment]
Removed .fadctorypath file and tested with WDT and command line.

File: README.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ Once the server is running, the application will be available under [http://loca
 
 ## Running with Maven
 
-This project can be built with [Apache Maven](http://maven.apache.org/). The project uses [Liberty Maven Plug-in][] to automatically download and install Liberty with Java EE7 Web Profile runtime from the Maven Central. Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server. 
+This project can be built with [Apache Maven](http://maven.apache.org/). The project uses [Liberty Maven Plug-in][] to automatically download and install Liberty with Java EE7 runtime from the Maven Central. Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server.
```
Comment (patricktiu) [inline README.md]: I would recommend to use the name `Liberty with Java EE7 Full Platform runtime`

File: pom.xml (hunk 1 above)
Comment (patricktiu) [inline pom.xml]: Execution of install-apps goal is included in the parent pom.  Wouldn't it copy the application to the apps folder?

**Comment by patricktiu** [comment]
Did you try to run the modified sample in WDT?  There are a few errors in eclipse.

File: README.md (hunk 4 above)
Comment (josephca) [inline README.md]: Fixed.

File: pom.xml (hunk 1 above)
Comment (josephca) [inline pom.xml]: I added install-apps goal instead.

**Comment by josephca** [comment]
As we discussed, it might be an issue with your workspace. I don't see any errors in eclipse.

**Comment by patricktiu** [comment]
We should use Maven standard directory layout for the project without resource directory customization in the war plugin.

``` xml
                 <plugin>
        		<groupId>org.apache.maven.plugins</groupId>
        		<artifactId>maven-war-plugin</artifactId>
        		<version>2.6</version>
        		<configuration>
					<failOnMissingWebXml>false</failOnMissingWebXml>
					<packagingExcludes>pom.xml</packagingExcludes>
         			<webResources>
           				<resource>
            			<!-- this is relative to the pom.xml directory -->
             			<directory>resources</directory>
            			</resource>
          			</webResources>
        		</configuration>
      		</plugin>
```

**Comment by patricktiu** [comment]
I have incorporated your latest update with my comments in `plugin-2.0-no-parent` branch.  If you are OK with the changes in this branch, you can accept PR #2

**Comment by josephca** [comment]
Merged by https://github.com/josephca/sample.jaxws.web/pull/2

**Pull request closed without merging**